runtime/internal/init_policy/policy.rego

```diff
@@ -0,0 +1,16 @@
+package init
```

Comment:
This is the current init policy, it is provided as an example. It makes use of all features: messages, dismissible errors and fatal errors.